src/components/InputBar/InputBar.jsx

```diff
@@ -23,9 +20,7 @@ const mapDispatchToProps = dispatch => ({
 
 });
 
-const InputBar = connect(
+export default connect(
```

Comment:
This is an example of me refactoring to a shorter export situation. It may be a case of being less readable however, so let me know if you want me to roll back that change.